contributors.yml

```diff
@@ -47,6 +47,7 @@
 - bhbs
 - bilalk711
 - bobziroll
+- bravo-kernel
```

Comment:
Seems like `main` and `dev` are not entirely in sync, by basing this off of `main` I'm merging this name into contributors. Should I remove it?

Reply:
Yep, this should be removed, and the comment change too.

Sorry, I missed this comment when replying earlier.